client/app/startup/TopbarApp.js

```diff
@@ -20,6 +20,6 @@ export default (props, marketplaceContext) => {
     'sign_up',
   ], { locale });
 
-  const combinedProps = Object.assign({}, props, { marketplaceContext, routes });
+  const combinedProps = Object.assign({}, props, { routes });
```

Comment:
Actually I tested this and it works, but I probably should remove the marketplace parameter from the call on CLP side